app/src/androidTest/java/com/android/sample/ui/activity/CreateActivityScreenTest.kt

```diff
@@ -1,99 +1,197 @@
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputDateCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputDateCreate").assertIsDisplayed()
+  }
 
-    @Test
-    fun createActivityScreen_displaysDateField() {
-        composeTestRule.setContent {
-            CreateActivityScreen(mockViewModel, mockNavigationActions)
-        }
-        composeTestRule.onNodeWithTag("inputDateCreate").assertExists()
-        composeTestRule.onNodeWithTag("inputDateCreate").assertIsDisplayed()
+  @Test
+  fun createActivityScreen_displaysPriceField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputPriceCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputPriceCreate").assertIsDisplayed()
+  }
 
-    }
+  @Test
+  fun createActivityScreen_displaysPlacesLeftField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputPlacesCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputPlacesCreate").assertIsDisplayed()
+  }
+
+  @Test
+  fun createActivityScreen_displaysLocationField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputLocationCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputLocationCreate").assertIsDisplayed()
+  }
+
+  @Test
+  fun attendeeRowDisplaysCorrectInformation() {
+    val attendee = SimpleUser("John", "Doe", 30)
+    val attendees = listOf(attendee)
 
-    @Test
-    fun createActivityScreen_displaysPriceField() {
-        composeTestRule.setContent {
-            CreateActivityScreen(mockViewModel, mockNavigationActions)
+    composeTestRule.setContent {
+      Row(
+          modifier = Modifier.padding(8.dp).background(Color(0xFFFFFFFF)),
+      ) {
+        Text(
+            text = attendees[0].name,
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Spacer(modifier = Modifier.width(8.dp))
+        Text(
+            text = attendees[0].surname,
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Spacer(modifier = Modifier.width(8.dp))
+        Text(
+            text = attendees[0].age.toString(),
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Button(
+            onClick = { /* no-op */},
+            modifier = Modifier.width(40.dp).height(40.dp).testTag("removeAttendeeButton"),
+        ) {
+          Icon(
+              Icons.Filled.PersonRemove,
+              contentDescription = "remove attendee",
+          )
```

Comment:
Why do you rewrite the code here explicitly instead of calling the original code ? Doesn't it go against the purpose of testing ?